includes/classes/Indexable/Post/Post.php

```diff
@@ -742,37 +742,33 @@ private function get_term_orders( $object_id ) {
 	}
 
 	/**
-	 * Prepare post meta to send to ES
+	 * Chekcs if meta key is allowed
 	 *
+	 * @param string  $meta_key meta key to check
 	 * @param WP_Post $post Post object
-	 * @since 0.1.0
-	 * @return array
+	 * @since 3.6.6
```

Comment:
The current EP version upstream for develop is 4.3.0:
```suggestion
	 * @since 4.3.0
```